### PR TITLE
Adding https://github.com/tektoncd/operator/issues/304 as test case

### PR DESCRIPTION
**Pull request by VeereshAradhya**
Adding https://github.com/tektoncd/operator/issues/304 as test case

### CODE REVIEW

**Review by ppitonak — APPROVED** [review]

**Review by vdemeester — APPROVED** [review]
/lgtm

File: specs/triggers/eventlistener.spec [hunk 1]
```diff
@@ -378,3 +378,44 @@ Steps:
...
+  * Mock post event to "github" interceptor with event-type "push", payload "testdata/push.json", with TLS "true"
+  * Assert eventlistener response
+  * Verify pipelinerun
+    |S.NO|pipeline_run_name  |status     |check_lable_propagation|
+    |----|-------------------|-----------|-----------------------|
+    |1   |simple-pipeline-run|successfull|no                     |
+  * Create
+    |S.NO|resource_dir                                                          |
+    |----|----------------------------------------------------------------------|
+    |1   |testdata/triggers/triggertemplate/triggertemplate-2.yaml              |
```
Comment (savitaashture) [inline specs/triggers/eventlistener.spec:412]: can't we use same triggertemplate :thinking:

Comment (VeereshAradhya) [reply specs/triggers/eventlistener.spec:412]: The previous triggertemplate had the pipelinerun name defined as `simple-pipeline-run`, so I created a new one

Comment (savitaashture) [reply specs/triggers/eventlistener.spec:412]: oh okok got it :+1:

**Review by savitaashture — COMMENTED** [review]
/lgtm

**Pull request merged**